dotnet/src/IntegrationTests/Connectors/Memory/CosmosMongoDB/CosmosMongoVectorStoreTests.cs

```diff
@@ -10,6 +10,8 @@ namespace SemanticKernel.IntegrationTests.Connectors.CosmosMongoDB;
 [Collection("CosmosMongoCollection")]
 [DisableVectorStoreTests(Skip = "Azure CosmosDB MongoDB cluster is required")]
 public class CosmosMongoVectorStoreTests(CosmosMongoVectorStoreFixture fixture)
+#pragma warning disable CA2000 // Dispose objects before losing scope
     : BaseVectorStoreTests<string, CosmosMongoHotel>(new CosmosMongoVectorStore(fixture.MongoDatabase))
```

Comment:
This warning is really annoying: we create an instance of `IDisposable` type here, so it says that we should dispose it before losing scope. The tooling does not recognize that the base class owns it. I could either add the pragma or change the `ctor` to accept a factory method and invoke it in the base class ctor

Reply:
Sure. I wouldn't spend any time on this though, as it's just test infra at the end of the day.